packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -0,0 +1,62 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SearchSuggestions = ({ autoCompleteHits, enterpriseSlug, handleViewAllClick }) => (
+  <div className="suggestions" data-testid="suggestions">
+    <div>
+      <div className="mb-2 ml-2 mt-1 font-weight-bold suggestions-section">
+        Courses
+      </div>
+      {
+                autoCompleteHits.filter(hit => hit.content_type === 'course')
+                  .slice(0, 3)
+                  .map((hit) => (
+                    <Link to={`/${enterpriseSlug}/course/${hit.key}`} key={hit.title} className="suggestion-item" style={{ whiteSpace: 'pre-wrap' }}>
+                      <div style={{ display: 'flex' }}>
+                        {/* eslint-disable-next-line no-underscore-dangle, react/no-danger */}
+                        <div dangerouslySetInnerHTML={{ __html: hit._highlightResult.title.value }} />
+                        <div className="badge badge-light ml-3 font-weight-light " style={{ lineHeight: '1.5' }}>
+                          {hit.key && hit.key.split('+')[0]}
```

Comment:
Aren't we supposed to show the complete course/program key here?

Reply:
this is basically to fetch name of authoring organization embedded in key, such as "HarvardX+CS50W", we would need only HarvardX from here. so i split this and choose relevant part.